### ISSUE TITLE
Each build have to download osx-64.zip to ./cache ?

### ISSUE THREAD

**Issue by liyaodong** [nwjs/nw-builder#233]
Every time build , it will force download `nwjs` latest version to cache , and every time build I can't success .

```
$ grunt nodewebkit
Running "nodewebkit:src" (nodewebkit) task
Latest Version: v0.12.2
Using v0.12.2
Create cache folder in /Users/dongdong/code/nodejs/checkin/cache/0.12.2
Downloading: http://dl.nwjs.io/v0.12.2/nwjs-v0.12.2-osx-x64.zip
  downloading [====================] 100% 0.0s

Create release folder in /Users/dongdong/code/nodejs/checkin/builds/gpk_checkin/osx64
Fatal error: ENOENT: no such file or directory, lstat '/Users/dongdong/code/nodejs/checkin/cache/0.12.2/osx64/nwjs.app'
```

I try to manually `mkdir` the folder , and the error still appear . I try to grunt and gulp module and the command line for `node-webkit-builder` , all failure .
my `Gruntfile.js`
```
module.exports = function(grunt) {
grunt.initConfig({
    nodewebkit: {
        options: {
            platforms: ['osx64'],
            buildDir: './builds',
            macZip: false
        },
        src: ['./App/**/*']
    }
});
grunt.loadNpmTasks('grunt-node-webkit-builder');
};

```

**Comment by adam-lynch** [comment]
I think this is resolved by #337 and version 2.2.1

**Comment by adam-lynch** [comment]
Thanks by the way!
